adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -370,10 +414,15 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 		}
 
 		routeP := createRoute(genRouteCreateParams(&mgwSwagger, resource, vHost, resourceBasePath, clusterNameProd,
-			clusterNameSand, operationalReqInterceptors, operationalRespInterceptorVal, organizationID))
+			clusterNameSand, operationalReqInterceptors, operationalRespInterceptorVal, organizationID, false))
+		if apiLevelbasePathSand != "" || resourceBasePathSandAvailable {
```

Comment:
better to add a comment explaining the order of the sand route, it should always come before prod route, right?

Reply:
yes, added a comment explaining the importance of order of the routes